bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -3520,7 +3520,7 @@ spec:
     name: Dell Technologies
     url: https://github.com/dell/csm-operator
   relatedImages:
-  - image: docker.io/dellemc/dell-csm-operator:v1.4.1
+  - image: docker.io/dellemc/dell-csm-operator:v1.4.2
```

Comment:
Version should be 1.5.0

Reply:
@panigs7 why is it 1.4.1 in main? When/how does this get updated? There's a lot of ourdated driver versions and stuff in there as well. I don't think this is an issue for this PR to solve. We will revert to 1.4.1 as it currently is in main.

Reply:
@panigs7 undid these changes, please re-review